tests/workbenches/conftest.py

```diff
@@ -0,0 +1,309 @@
+from __future__ import annotations
```

Comment:
I think the majority of these classes could be avoided if you instead used the wrapper library https://github.com/RedHatQE/openshift-python-wrapper

Reply:
used ocp_resources more, thanks